jep/200/README.adoc

```diff
@@ -212,8 +226,10 @@ this is simply the cost of having a tighter security policy.
 To ameliorate the risk we can check automated test results against the patched core,
 specifically scanning for the term `class-filter` which appears in logs whenever a violation is encountered.
 Some runs of `acceptance-test-harness` (ATH) were already performed in this mode.
-`plugin-compat-tester` (PCT) was also run against an array of plugins (including those bundled in CloudBees products);
-unfortunately the Jenkins project currently has no maintained CI job running PCT against all plugins suggested by the setup wizard.
+`plugin-compat-tester` (PCT) was also run against an array of plugins.
+See the list in _Appendix A_ for more details.
+// oleg-nenashev: I will try to solve some bits of it before approving this JEP
+Unfortunately the Jenkins project currently has no maintained CI job running PCT against all plugins suggested by the setup wizard.
```

Comment:
This line should be moved to the appendix I suppose, or the appendix should just be inlined here.